java-samples/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <name>Google Cloud Client Library for Java Samples</name>
+    <artifactId>google-cloud-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>1.38.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-java:current} -->
```

Comment:
Can you make the version as "0.1.0-SNAPSHOT" without Release Please annotations? With that, we ensure that the module won't be published to Maven Central.

Reply:
Done, that's a good idea.